site/content/en/_index.html

```diff
@@ -22,33 +22,39 @@
 {{% blocks/lead color="primary" %}}
 <strong>What is Agones?</strong>
 
-Agones is an open source project that is built to be a batteries-included, dedicated game server scaling and
-orchestration platform. It is built as a native extension of [Kubernetes](https://kubernetes.io), with the flexibility you need to tailor
-it to the requirements of your multiplayer game. Anywhere you can run Kubernetes, you can run Agones.
-
+An open source, batteries-included, mulitplayer dedicated game server scaling and
+orchestration platform that can run anywhere [Kubernetes](https://kubernetes.io) can run.
 {{% /blocks/lead %}}
 
 {{< blocks/section color="dark" >}}
-{{% blocks/feature icon="fas fa-file-code" title="Fleets of GameServers" url="./docs/getting-started/" %}}
-Define Fleets of GameServers, autoscaling and more, through YAML or the Kubernetes API.
+{{% blocks/feature icon="fas fa-file-code" title="Orchestrate Game Servers" url="./docs/getting-started/" %}}
+Define and manage warm groups of ready gameservers through YAML configuration or API calls.
```

Comment:
nit: s/warm/pre-spun/ perhaps ?

Reply:
Maybe just drop "warm" all together? We also say "Ready" in this sentence. Wdyt?